cms/templates/cms/home_page.html

```diff
@@ -187,6 +142,52 @@ <h4 class="program-title" aria-describedby="program-{{ program.id }}-description
         </ul>
       </section>
 
+      <div class="row reasons-section">
+
+      <div class="centered-content">
+
+        <h3>Why should I enroll?</h3>
+        <div class="enroll-reason col-md-4">
+          <div class="reasons-images-wrap">
+             <img src="{% static 'images/online.png' %}" alt="">
+          </div>
+
+          <h5 class="upper-case">Convenient &amp; Flexible</h5>
```

Comment:
`Why should I enroll?` is in an `<h3>` tag. Therefore, these reasons should all be in `<h4>` tags, not `<h5>` tags.